components/src/core/dropdown-toolbar/dropdown-toolbar.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, ViewEncapsulation, Input, ViewChild, ElementRef } from '@angular/core';
+import { isEmptyView } from '../../utils/utils';
+import { MatMenuTrigger } from '@angular/material/menu';
+
+@Component({
+    selector: 'pxb-dropdown-toolbar',
+    encapsulation: ViewEncapsulation.None,
+    template: `
+        <mat-toolbar fxLayout="row" class="pxb-dropdown-toolbar">
+            <div
+                #icon
+                class="pxb-dropdown-toolbar-icon-wrapper"
+                [class.pxb-dropdown-toolbar-no-icon]="isIconEmpty(iconEl)"
+            >
+                <ng-content select="[pxb-nav-icon]"></ng-content>
+            </div>
+            <div *ngIf="title" class="pxb-dropdown-toolbar-text-content-container">
+                <div class="pxb-dropdown-toolbar-title mat-title">{{ title }}</div>
+                <div
+                    *ngIf="subtitle"
+                    class="pxb-dropdown-toolbar-subtitle-container mat-subheading-2"
+                    [matMenuTriggerFor]="dropdownToolbarMenu"
+                    #menuTrigger="matMenuTrigger"
+                >
+                    <span class="pxb-dropdown-toolbar-subtitle mat-subheading-2">{{ subtitle }}</span>
+                    <mat-icon [class.rotated-dropdown-arrow]="menuTrigger.menuOpen">arrow_drop_down</mat-icon>
+                </div>
+            </div>
+            <pxb-spacer></pxb-spacer>
```

Comment:
May want to leave out the spacer and let the user pass that in with the rest of their ng-content if they want it.